compliance/compliance.go

```diff
@@ -2013,3 +2018,51 @@ func ForwardReferencesDisallowed(c *fluent.GRIBIClient, t testing.TB, _ ...TestO
 			AsResult(),
 		chk.IgnoreOperationID())
 }
+
+// AddNHGReferencingToUknownNH tests a case where a nexthop group references a nexthop IP that does not Resolve.
+func AddNHGReferencingToUknownNH(c *fluent.GRIBIClient, wantACK fluent.ProgrammingResult, t testing.TB, _ ...TestOpt) {
+	defer flushServer(c, t)
+	ops := []func(){
+		func() {
+			c.Modify().AddEntry(t,
+				fluent.NextHopEntry().WithNetworkInstance(defaultNetworkInstanceName).WithIndex(1).WithIPAddress("192.0.2.30"),
+				fluent.NextHopGroupEntry().WithNetworkInstance(defaultNetworkInstanceName).WithID(1).AddNextHop(1, 1))
+
+		},
+	}
+	t.Log("Sleeping for 100 seconds to allow the DUT to process the new configuration")
```

Comment:
When you move this to featureprofiles, you should use an `Await` here rather than a very long sleep.